src/components/hooks/useTheme.ts

```diff
@@ -0,0 +1,17 @@
+const capitalize = require('lodash/capitalize');
+
+import { useTheme } from '../../theme';
+import { FormtronComponentVariant } from '../../types';
+
+const useProp = (prop: string) => (variant?: FormtronComponentVariant) => {
```

Comment:
Ummm,  could be a part UI-Kit, perhaps?

Reply:
might be premature to include it. I'm not sure what theming will look like in the near future.

Reply:
Yep, just floated an idea. Might be useful in future.